### ISSUE TITLE
Post property "post_status" doesn't work properly

### ISSUE THREAD

**Issue by guillermomf2001** [vaakash/git-it-write#13]
When I make a `git push` to my repository I have an HTTP Error 500 in the webhook log due to the Yaml `post_status: draft` configuration in the .md post file. I tried the other values to `post_status` and doesn't work. The only ones that works right is `post_status: publish`

```
---

title: Title of this post
menu_order: 1
post_status: draft  # <-- it does not work!! (HTTP Error 500)
post_excerpt: Excerpt of this post.
taxonomy:
    category:
        - Javascript
    post_tag:
        - Javascript-config
        - VSCode

---

# Title post

The rest of the content...
```

**Comment by vaakash** [comment]
Hi @ZonaGuillermo,

I'll check this issue and update.

Thanks,
Aakash

**Comment by vaakash** [comment]
This is addressed in v1.7. Marking this as closed.
